src/main/resources/digital/slovensko/autogram/server/server.yml

```diff
@@ -527,12 +527,41 @@ components:
+          description: Optional XML schema used to validate the signing document and to compute digest used in "UsedXSDReference" in "DigestValue" attribute inside created XML Datacontainer. Format (plaintext or base64) is dictated by `payloadMimeType`. Is overridden by autoLoadEform.
+
+        schemaIdentifier:
+          type: string
+          example: http://schemas.gov.sk/form/App.GeneralAgenda/1.9/form.xsd
+          description: Optional identifier of the XML schema. The value is used in "UsedXSDReference" field inside created XML Datacontainer. Schema must be provided with this parameter. Is overridden by autoLoadEform. Default value is "http://schemas.gov.sk/form/<form-idnetifier>/<version>/form.xsd".
 
         transformation:
           type: string
           example: '<?xml version="1.0"?><xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"><xsl:template match = "/"><h1><xsl:value-of select="/Document/Title"/></h1></xsl:template></xsl:stylesheet>'
-          description: Optional XML transformation used to . Format is dictated by `payloadMimeType`. Is overridden by autoLoadEform.
+          description: Optional XML transformation used to present the signing document to user and to compute digest used in "UsedPresentationSchemaReference" in "DigestValue" attribute inside created XML Datacontainer. Format (plaintext or base64) is dictated by `payloadMimeType`. Is overridden by autoLoadEform.
+
+        transformationIdentifier:
+          type: string
+          example: http://schemas.gov.sk/form/App.GeneralAgenda/1.9/form.xslt
+          description: Optional identifier of the XML transformation. The value is used in "UsedPresentationSchemaReference" field inside created XML Datacontainer. Transformation must be provided with this parameter. Is overridden by autoLoadEform. Default value is "http://schemas.gov.sk/form/<form-idnetifier>/<version>/form.xslt".
+
+        transformationLanguage:
+          type: string
+          example: sk
+          description: Optional language of the XML transformation. The value is used in "UsedPresentationSchemaReference" in "Language" attribute inside created XML Datacontainer. Transformation must be provided with this parameter. Is overridden by autoLoadEform. Default value is "sk".
+
+        transformationMediaDestinationTypeDescription:
+          type: string
+          enum:
+            - TXT
+            - HTML
+            - XHTML
+          example: TXT
+          description: Optional media destination type description of the XML transformation. The value is used in "UsedPresentationSchemaReference" in "MediaDestinationTypeDescription" attribute inside created XML Datacontainer. Transformation must be provided with this parameter. Overrides value of output method in XSLT which is used by default. Is overridden by autoLoadEform.
+
+        transformationTargetEnvironment:
```

Comment:
Toto som ešte nikdy nevidel (aj keď v štandarde to ako optional je) a aj Szilva hovoril, že sa to nepoužíva a nevidel to ešte v praxi použité.

Reply:
Toto overit so stefanom, ze ako toto meni spravanie vlastne a ci vobec. ak to nemeni nic, len to treba vlozit do xdc tak to pridajme.